pkg/library/kuadrant/apimachinery_status_conditions.go

```diff
@@ -107,14 +107,12 @@ func AcceptedCondition(p Policy, err error) *metav1.Condition {
 func EnforcedCondition(policy Policy, err PolicyError, allSubresourcesReady bool) *metav1.Condition {
 	// Enforced
 	message := fmt.Sprintf("%s has been successfully enforced", policy.Kind())
-	status := metav1.ConditionTrue
 	if !allSubresourcesReady {
 		message = fmt.Sprintf("%s has been partially enforced", policy.Kind())
-		status = metav1.ConditionFalse
 	}
 	cond := &metav1.Condition{
 		Type:    string(PolicyConditionEnforced),
-		Status:  status,
+		Status:  metav1.ConditionTrue,
 		Reason:  string(PolicyReasonEnforced),
 		Message: message,
 	}
```

Comment:
From what i understand this was the important bit to revert from https://github.com/Kuadrant/kuadrant-operator/pull/677/files#diff-c62f06c8c31cd4ef9b0a3f839225773a42ea10c75dff00563b42b413ec425107, but we still need this https://github.com/Kuadrant/kuadrant-operator/pull/677/files#diff-06c8a42ee2952e17c9b90724b17324a7dfb178dbea00e606ce3c6eb62b3bd73eR118, so i think this is fine.

Reply:
Yes, just the `Status` is the problem. The rest should stay